### ISSUE TITLE
학습 시나리오 1 테스트

### ISSUE THREAD

**Issue by TukBang** [TukBang/Pet-care-app#2]
Model 1: A1 ~ A6 (6-CLASS), EfficientNetV2_s
Preprocessing: ratio-and-zero-padding, interpolation: LANCZOS4
batch_size: 64, 128
optimizer = Adam
learning rate: 1e-4
criterion = CrossEntropyLoss

보간법에 의한 CNN 학습 차이 확인

이전, 보간법이 BILINEAR일 때 Train: 95%, valid: 59.8%

**Comment by TukBang** [comment]
각 5600X PC, 5900X PC에서 관심영역 이미지 데이터셋을 생성할 때 관심영역의 크기 조건이 다르게 생성되어
데이터셋 내부 개수 1% 차이가 존재함

이에 따라, 동일한 실험환경 구성을 하기 위해 조건들 갖추어야 하며 현재 진행 중인 모델은 대략적인 평가만 진행

**Comment by TukBang** [comment]
precision    recall  f1-score   support

          A1       0.59      0.56      0.57      1087
          A2       0.62      0.67      0.65      1331
          A3       0.55      0.69      0.61      1317
          A4       0.63      0.63      0.63      1092
          A5       0.63      0.40      0.49      1056
          A6       0.84      0.81      0.83       762

    accuracy                           0.62      6645

A1(구진, 플라크), A5(미란, 궤양)이 다른 레이블에 비해 낮아보이는 수치를 가지고 있어,
다음 시나리오로 진행

batch_size 64보다 128이 더 높은 수치를 기록하여 128만 기록함

**Comment by TukBang** [comment]
![AUC-ROC](https://user-images.githubusercontent.com/121234327/226327372-e2bfb5de-26bc-4c78-a5e9-1597c7a1a492.png)

AUC per class:
[0.8540246983139745, 0.8725201592436745, 0.8659064816524953,
0.8819815081970674, 0.8260698436322431, 0.961470793330844]

**Comment by TukBang** [comment]
![graph](https://user-images.githubusercontent.com/121234327/226536457-750544bd-78da-47bb-a449-41ea363e7099.png)

epoch 138
